### PR TITLE
feat(sbt): Group dependencies by their variable names

### LINKED ISSUE #6706: MR grouping not working with sbt
**What Renovate type are you using?**

Self hosting Renovate.

**Describe the bug**

Given the following `build.sbt` file:
```scala
val cucumberVersion = "6.2.0"

libraryDependencies ++= Seq(
  "io.cucumber" %% "cucumber-scala" % cucumberVersion % Test,
  "io.cucumber" % "cucumber-junit" % cucumberVersion % Test,
  "io.cucumber" % "cucumber-picocontainer" % cucumberVersion % Test
)
```

When RenovateBot runs, it correctly detects and suggest to update the version in `cucumberVersion` variable but the summary of the MR (and the title as well) only mentions one library update:

image

I've seen in Maven projects that the 3 libraries are grouped as a single MR and I would expect the same behavior for sbt projects.

**Relevant debug logs**

N/A

**To Reproduce**

Use sample above.

**Additional context**

N/A

### PR DESCRIPTION
**Pull request by zharinov**
Closes #6706

Example: https://github.com/zharinov/renovate-issue-6706/pull/1

### CODE REVIEW

**Review by viceice — DISMISSED** [review]

**Comment by zharinov** [comment]
Need to separate between different sbt files

File: lib/manager/sbt/extract.ts [hunk 1]
```diff
@@ -176,6 +176,10 @@ function parseDepExpr(
     currentValue,
   };
 
+  if (variables[rawVersion]) {
+    result.groupName = `${rawVersion} for ${groupId}`;
```
Comment (zharinov) [inline lib/manager/sbt/extract.ts:180]: It's hard to separate by package file, but maybe it's good enough to group just with single `groupId`

Comment (gaeljw) [reply lib/manager/sbt/extract.ts:180]: IMHO it's good enough to group by `groupId` ;)

Comment (zharinov) [reply lib/manager/sbt/extract.ts:180]: @viceice @rarkins What grouping criteria would you prefer?

Comment (viceice) [reply lib/manager/sbt/extract.ts:180]: `groupId` sounds good to me

Comment (zharinov) [reply lib/manager/sbt/extract.ts:180]: Okay, so I'm leaving it as-is, i.e. `variable + groupId`

**Review by rarkins — APPROVED** [review]

File: lib/manager/sbt/extract.ts (hunk 1 above)
Comment (viceice) [reply lib/manager/sbt/extract.ts:180]: 🤔 ok

**Review by viceice — APPROVED** [review]

**Pull request merged**